### PR TITLE
Add support for custom path

### LINKED ISSUE #23: Feature request: Set path through environment variables
At the moment, the path is hardcoded "/mapcache/". It would be nice if it would be possible to change that through environment variables.

### PR DESCRIPTION
**Pull request by yjacolin**
Default path is mapcache for history compatibility. Fix #23

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,7 @@ FROM debian:buster
 MAINTAINER Camptocamp "info@camptocamp.com"
 
 ENV MAPCACHE_VERSION=branch-1-10 \
+    MAPCACHEPATH=mapcache \
```
Comment (fredj) [inline Dockerfile]: minor but could the name be changed to `MAPCACHE_PATH`?

Comment (yjacolin) [reply Dockerfile]: done, thanks

**Review by fredj — APPROVED** [review]

**Comment by yjacolin** [comment]
@fredj  do we need to do something to backport this on master?

**Comment by fredj** [comment]
> @fredj do we need to do something to backport this on master?

I'm not sure how it's done in this project, any idea @sbrunner ?

**Pull request merged**